### PR TITLE
Added Vela Exchange to perpetual_trades

### PR DESCRIPTION
**Pull request by ARDev097**
Added vela_exchange perpetual trade data.

### CODE REVIEW

File: models/unidex/optimism/unidex_optimism_trades.sql [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    {% endif %}
+LEFT JOIN {{ source('prices', 'usd') }} p_sold
+    ON p_sold.minute = date_trunc('minute', dexs.block_time)
+    AND p_sold.contract_address = dexs.token_sold_address
+    AND p_sold.blockchain = 'optimism'
+    {% if not is_incremental() %}
+    AND p_sold.minute >= TIMESTAMP '{{project_start_date}}'
+    {% endif %}
+    {% if is_incremental() %}
+    AND p_sold.minute >= date_trunc('day', now() - interval '7' day)
```
Comment (Hosuke) [inline models/unidex/optimism/unidex_optimism_trades.sql:116]: Hi @ARDev097, you can use the `incremental_predicate` in incremental filters in future:
```suggestion
    AND {{ incremental_predicate('p_sold.minute') }}
```

**Comment by ARDev097** [comment]
Hello @Hosuke , I have made the requested changes. Please let me know if anything else is required from my end.

File: models/perpetual/perpetual_trades.sql [hunk 2]
```diff
@@ -21,6 +21,8 @@
 ,ref('hubble_exchange_avalanche_c_perpetual_trades')
 ,ref('gmx_perpetual_trades')
 ,ref('tigris_perpetual_trades')
+,ref('unidex_perpetual_trades')
+,ref('vela_exchange_perpetual_trades')
```
Comment (Hosuke) [inline models/perpetual/perpetual_trades.sql:25]: This PR may come after #5067

File: models/vela_exchange/vela_exchange_perpetual_trades.sql [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{{ config(
+    
+    alias = 'perpetual_trades',
+    schema = 'vela_exchange_perpetual',
```
Comment (Hosuke) [inline models/vela_exchange/vela_exchange_perpetual_trades.sql:4]:
```suggestion
    schema = 'vela_exchange',
    alias = 'perpetual_trades'
```

File: models/vela_exchange/base/vela_exchange_base_perpetual_trades.sql [hunk 4]
```diff
@@ -0,0 +1,129 @@
+{{ config(
+    alias = 'perpetual_trades',
+    schema = 'vela_exchnage_base_perpetual',
```
Comment (Hosuke) [inline models/vela_exchange/base/vela_exchange_base_perpetual_trades.sql:3]:
```suggestion
    alias = 'perpetual_trades',
    schema = 'vela_exchnage_base',
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @ARDev097 

[Sample vela_exchange_base_perpetual_trades](https://dune.com/queries/3340073)

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**